### PR TITLE
Removed updates 9001, 9002, 9003 per WS2-1209

### PR DESCRIPTION
**Pull request by chill5-0**
WS2-1209 deletes 9001 and 9003 -- I'm adding 9002 for this PR. Let me know if 9002 should remain.

### CODE REVIEW

File: webspark_blocks.install [hunk 1]
```diff
@@ -2,52 +2,6 @@
...
-    $block->delete();
-  }
-  // Remove the block type basic.
-  $storage = \Drupal::entityTypeManager()->getStorage('block_content_type');
-  $block_type = $storage->load('basic');
-  if ($block_type) {
-    $block_type->delete();
-  }
-  // Revert all the configs.
-  _webspark_blocks_revert_module_config();
```
Comment (mlsamuelson) [inline webspark_blocks.install:41]: 9002 should remain, but line 41  which does a revert config should be removed:
```
_webspark_blocks_revert_module_config();
```
My rationale is that we want to maintain consistency with removing the basic blocks, but we need to not have a config revert happen until after the background color updates have occurred. 9002 without the config revert line should accomplish that.

**Review by mlsamuelson — CHANGES_REQUESTED** [review]
See comment related to retaining most of 9002.

File: webspark_blocks.install (hunk 1 above)
Comment (chill5-0) [reply webspark_blocks.install:41]: Sounds good! New commit added with the changes.

**Review by mlsamuelson — APPROVED** [review]
Thanks @chill5-0 ! Approved.

**Pull request merged**